kani-compiler/src/codegen_cprover_gotoc/context/current_fn.rs

```diff
@@ -104,6 +112,11 @@ impl<'tcx> CurrentFnCtx<'tcx> {
 
 /// Utility functions
 impl CurrentFnCtx<'_> {
+    /// Is the current function from the `std` crate?
+    pub fn is_std(&self) -> bool {
```

Comment:
Will this only be true for generic `std`/`core` functions, i.e. that will be monomorphized? I'm assuming non-generic `std` functions are not codegen unless we're compiling the `std` crate itself?

Reply:
I think this also applies to std functions that get inlined in.